packages/block/test/header.spec.ts

```diff
@@ -150,16 +144,13 @@ describe('[Block]: Header functions', () => {
     headerArray[14] = new Uint8Array(8) // nonce
 
     let header = createBlockHeaderFromBytesArray(headerArray, { common })
-    assert.isTrue(Object.isFrozen(header), 'block should be frozen by default')
+    assert.isFrozen(header, 'block should be frozen by default')
 
     header = createBlockHeaderFromBytesArray(headerArray, { common, freeze: false })
-    assert.ok(
-      !Object.isFrozen(header),
-      'block should not be frozen when freeze deactivated in options',
-    )
+    assert.isNotFrozen(header, 'block should not be frozen when freeze deactivated in options')
   })
 
-  it('Initialization -> createWithdrawalFromBytesArray() -> error cases', () => {
+  it('Initialization -> createWithdrawalFromBytesArray() -> error cases', async () => {
```

Comment:
Interesting, `async` is added but no `await` was added in this code block :thinking: Would also assume that a linter would complain here (maybe it warns?). No need to remove it but I'd really expect a linter to remove this when the "npm run lint:fix" is ran.